app/src/main/res/layout/continue_interaction_item.xml

```diff
@@ -46,14 +50,19 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
 
-    <Button
-      android:id="@+id/continue_button"
+    <org.oppia.android.app.customview.ContinueButtonView
+      android:id="@+id/animation_continue_button"
       style="@style/StateButtonActive"
-      android:layout_width="wrap_content"
+      android:layout_marginEnd="10dp"
+      android:gravity="center"
+      android:minWidth="96dp"
+      android:minHeight="36dp"
       android:onClick="@{(v) -> viewModel.handleButtonClicked()}"
       android:text="@string/state_continue_button"
+      android:textSize="14sp"
```

Comment:
Why this change?

Reply:
Covered in the thread above as part of "why are extra changes needed." Closing this to consolidate the thread.